k9mail/src/main/java/com/fsck/k9/activity/Accounts.java

```diff
@@ -1290,7 +1291,7 @@ private void onAbout() {
                               getString(R.string.app_revision_url) +
                               "</a>"))
         .append("</p><hr/><p>")
-        .append(String.format(getString(R.string.app_copyright_fmt), year, year))
+        .append(String.format(getString(R.string.app_copyright_fmt), Integer.toString(year), Integer.toString(year)))
```

Comment:
Hm, what is the process for such a change? Do I just update the English translation and the others will be retranslated through transifex? I wanted to avoid the work that minimal change :D